assets/src/edit-story/components/canvas/useCanvasSelectionCopyPaste.js

```diff
@@ -143,17 +155,75 @@ function useCanvasSelectionCopyPaste(container) {
         /**
          * Loop through all items in clipboard to check if correct type. Ignore text here.
          */
+        const possibleRetryFiles = [];
```

Comment:
All the same things here. This is obvious code duplication.

Reply:
Reverted to another PR